tests/debug_info_test.py

```diff
@@ -30,17 +32,77 @@
 import jax.custom_batching
 import jax.custom_derivatives
 import jax.custom_transpose
-from jax.errors import UnexpectedTracerError
+from jax.experimental import pallas as pl
```

Comment:
It looks like this needs an update in `tests/BUILD` adding `jax/experimental` (I think?) as a dep.